requirements.txt

```diff
@@ -5,6 +5,9 @@ Flask-SQLAlchemy==2.1
 MarkupSafe==0.23
 MySQL-python==1.2.5
 psycopg2==2.6.2
+Flask-SQLAlchemy==2.1
+flask-cors>=3.0.2
+argparse==1.2.1
```

Comment:
is argparse needed for cors?  same for Flask-SQLAlchemy?  They seem to be used for other things.